crates/bevy_pbr/src/render/mesh_view_types.wgsl

```diff
@@ -85,3 +85,12 @@ struct ClusterOffsetsAndCounts {
     data: array<vec4<u32>>,
 };
 #endif
+
+struct Globals {
+    // The time since startup in seconds
+    time: f32,
+    // The delta time of the previous frame in seconds
```

Comment:
Hmm. 'of' may be technically correct, but it made my brain halt for a moment. Maybe 'since' would flow better for readers?